From: =?utf-8?q?Rog=C3=A9rio_Brito?= <rbrito@gmail.com>
Date: Fri, 12 Feb 2021 15:46:34 -0300
Subject: pdf.py: Make python shebang versioned.

---
 pdf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pdf.py b/pdf.py
index 9237141..b880b23 100755
--- a/pdf.py
+++ b/pdf.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 # Copyright 2006 Google Inc.
 # Author: agl@imperialviolet.org (Adam Langley)
 #
